static/js/modules/columns.js

```diff
@@ -506,6 +506,26 @@ var reports = {
   }
 };
 
+var loans = [
+  {
+    data: 'committee',
+    orderable: false,
+    className: 'all column--large',
+    render: function (data, type, row) {
```

Comment:
This `render` function should also build a link to the page, like the other tables do:
```js
      if (data) {
        return columnHelpers.buildEntityLink(
          data.name,
          helpers.buildAppUrl(['committee', data.committee_id]),
          'committee'
        );
      } else {
        return '';
      }
```